integration-cli/docker_cli_network_unix_test.go

```diff
@@ -135,6 +136,11 @@ func (s *DockerNetworkSuite) TestDockerNetworkCreateDelete(c *check.C) {
 	assertNwNotAvailable(c, "test")
 }
 
+func (s *DockerSuite) TestDockerNetworkDeleteNotExists(c *check.C) {
```

Comment:
I assumed he noticed this test was missing while touching/testing the networking API changes. I did a quick check and didn't notice this being tested.